contrib/helm/pganalyze-collector/templates/deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
       labels:
         {{- include "pganalyze-collector.selectorLabels" . | nindent 8 }}
     spec:
+      enableServiceLinks: false
```

Comment:
This adds a lot of service discovery variables, which are useless for most applications:

```
EnableServiceLinks indicates whether information about services should be injected into pod's environment variables, matching the syntax of Docker links. Optional: Defaults to true.
```

From https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#pod-v1-core

Reply:
I wonder if there would be a use case for this, if you had your Postgres running inside Kubernetes as well? (though even then I imagine the collector would actually have to read out the relevant env variables?)

Reply:
I think even for users trying to monitor PostgreSQL running in Kubernetes, it would be more likely that someone use a service DNS name instead of the IP address. We can move this to a default setting in values.yaml if we want to make it configurable?

Reply:
I think you're probably right, but maybe making it configurable doesn't hurt, just to avoid something having to fork the chart to customize this.

Reply:
Thinking through this again, given that this is mainly about the injection of environment variables, there isn't actually much benefit to allowing this to be configurable - since the collector doesn't use these variables. Thus I think we can actually leave this as `enableServiceLinks = false` in all cases (and revise only if we hear feedback that this needs to be configurable).